ppl/cmd/zar/index/add.go

```diff
@@ -0,0 +1,160 @@
+package index
```

Comment:
I see you changed the subcommand to `create`; would you please also rename this file to create.go, and adjust the `Add`, `AddCommand`, and related refs to `Create`?

Reply:
Shoot, my bad